step-definitions/core/core.js

```diff
@@ -1,10 +1,16 @@
 import { client } from 'nightwatch-api';
-import { When, Then } from 'cucumber';
+import { Given, When, Then } from 'cucumber';
 
 import { setupEvan } from '../../test-utils/test-utils.js';
 
 export const WAIT_TIME = 15 * 1000;
 
+Given(/^I open the path "([^"]+)"$/, async path => {
```

Comment:
isn't this the same as here? ui/ui-tests/step-definitions/standard-scenarios/browser.js

````
Given("I am on the path {string}", async (path) => {
  await client.url(`${ evan.baseUrl }${path}`);
  await client.pause(1000)
})
```

Reply:
well watched :D